README.md

```diff
@@ -21,49 +21,55 @@ API application template for Yii 3.
 
 ## Installation
 
-Creating a project:
+If you have composer locally, create a project:
 
 ```shell
 composer create-project yiisoft/app-api --stability=dev myproject
 cd myproject
 ```
 
-Install docker:
+If you don't have, fork the repository, clone it, then:
 
 ```shell
-docker-compose up -d
+cd myproject
+make composer update
 ```
 
-Enter into the container:
+## Running the app
+
+To run the app:
 
 ```shell
-docker-compose exec php bash
+make up
 ```
 
-Install packages:
+The application is available at `https://localhost`.
+
+
+To stop the app:
 
 ```shell
-composer install
```

Comment:
I'd keep plain php usage guide as is and add a new one for makefile+docker usage.